app/brave_generated_resources.grd

```diff
@@ -308,13 +308,25 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
         New Tor Connection For This Site
       </message>
       <message name="IDS_PROFILES_EXIT_TOR" desc="Button in the avatar menu bubble view for exiting the active Tor session.">
-        Exit Tor
+        Exit Private Window with Tor
+      </message>
+      <message name="IDS_PROFILES_EXIT_PRIVATE" desc="Button in the avatar menu bubble view for closing private window.">
+        Exit Private Window
       </message>
       <message name="IDS_TOR_PROFILE_NAME" desc="The name given to the Tor browsing profile. Displayed in the avatar menu bubble and button UI.">
-        Tor
+        Private Window with Tor
+      </message>
+      <message name="IDS_PRIVATE_PROFILE_NAME" desc="The name given to the private profile. Displayed in the avatar menu bubble and button UI.">
+        Private Window
+      </message>
+      <message name="IDS_TOR_AVATAR_BUTTON_LABEL" desc="The text label of tor avatar button.">
+        Private with Tor
+      </message>
+      <message name="IDS_TOR_AVATAR_BUTTON_TOOLTIP_TEXT" desc="The tooltip text of tor avatar button.">
+        This is private window with tor
```

Comment:
Missing " a" and capitalization of "Tor":

> This is **a** private window with **Tor**

Reply:
Fixed.

Reply:
Looks like it still has the old spelling? 
![image](https://user-images.githubusercontent.com/4733304/99828009-3e9e5600-2b0f-11eb-80f7-30083b4b472f.png)

Reply:
@bsclifton  Oops, something happened while rebasing. fixed again. Thanks for checking!